Lib/http/server.py

```diff
@@ -1301,6 +1304,9 @@ def test(HandlerClass=BaseHTTPRequestHandler,
                         default='HTTP/1.0',
                         help='conform to this HTTP version '
                              '(default: %(default)s)')
+    parser.add_argument('--content-type',  # parsed into content_type
```

Comment:
```suggestion
    parser.add_argument('--content-type',
```

We may assume that the reader already knows